pkg/clusterd/disk.go

```diff
@@ -113,6 +118,38 @@ func DiscoverDevices(executor exec.Executor) ([]*sys.LocalDisk, error) {
 	return disks, nil
 }
 
+func deviceMatchWithFilter(device string, filter, metaDevice string) bool {
+	if filter == listAllDevices || device == metaDevice {
+		return true
+	}
+
+	// We need to allow dm- devices (Kubernetes mountpoints) because dm- devices name are used as
+	// name pattern for meta device
+	pattern, err := regexp.Compile("dm")
```

Comment:
This should be in the first commit, rather than the commit with the CI timeouts. Also, need to check for err.